spells.cpp

```diff
@@ -199,11 +187,8 @@ void Game::ProcessInvisibility(Unit *u, parser::string_parser& parser, orders_ch
     if (u->castorders && u->castorders->spell == S_INVISIBILITY) {
         order = dynamic_cast<CastUnitsOrder *>(u->castorders);
     } else {
-        order = new CastUnitsOrder;
-        order->spell = S_INVISIBILITY;
-        order->level = 1;
         u->ClearCastOrders();
-        u->castorders = order;
+        u->castorders = new CastUnitsOrder(1);
```

Comment:
```suggestion
        order = new CastUnitsOrder(1);
        u->castorders = order; 
```

Reply:
This specific spell ended up not defining order when going through this pass which caused a crash.  This fixes that.  Other places didn't toss out the initialization.